packages/docs/docs/vision.md

```
@@ -8,7 +8,19 @@ title: Actual Budget Vision
 
 - take charge of their finances
 - find financial peace of mind
-- actively engage in dreaming big and achieving your goals
+- actively engage in dreaming big and achieving their goals
+
+### Primary Objectives
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)